docs/developers/developers-setup.md

```diff
@@ -177,8 +177,7 @@ To run Diffuse:
 diffuse
 ```
 
-_Note: The `diffuse` command can be used to launch Diffuse as a native Mac app_
-_that is installed into `/Applications/Diffuse.app`._
+_Note: The `diffuse` command can be used to launch Diffuse as a native Mac app that is installed into `/Applications/Diffuse.app`. To pass command-line arguments, run `diffuse_impl`. _
```

Comment:
Like I said in, https://github.com/MightyCreak/diffuse/pull/173#discussion_r1009338457, this is not entirely true since you *can* actually pass command-line args also to `diffuse`. 

If you wanna be more truth-full, write something like:

```
You can pass command-line arguments to `diffuse` but on macOS `diffuse` is a wrapper around macOS's `open` so you will not see the output. If you need to see the output, call `diffuse_impl` directly instead.
```

This might be good info but then perhaps rather in docs/users.md? I guess it would be good to create a TODO-Issue for this link to it in the doc.

Reply:
This is not for merging, rather a reminder that we might want look into fixing this if possible?

Reply:
Is this PR not for merging?

Reply:
Absolutely. I had hoped you could fix it right away, in this case an issue might have been unnecessary.

Reply:
Aha, gotcha. No, I don't know actually... I had a quick try but I don't know what makes `open` swallow stdout.